app/models/user.rb

```diff
@@ -10,11 +10,11 @@ class User < ApplicationRecord
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
   has_many :friendships
-  has_many :inverse_friendships, class_name: 'Friendship', foreign_key: 'friend_id'
+  # has_many :inverse_friendships, class_name: 'Friendship', foreign_key: 'friend_id'
```

Comment:
Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.